### ISSUE TITLE
V. 0.65.1 - can no longer add new Custom Emojis

### ISSUE THREAD

**Issue by earlmedina** [RocketChat/Rocket.Chat#10966]
### Description:

At version 0.65.1, no longer see the "+" to add new custom emojis. The UI appears to be missing this and so it is not possible to add new custo emojis.

### Server Setup Information:

* Version of Rocket.Chat Server: 0.65.1
* Operating System: CentOS 7, also Ubuntu 16.04
* Deployment Method(snap/docker/tar/etc): manual install on CentOS 7, snapd on Ubuntu
* Number of Running Instances: 
* DB Replicaset Oplog: 
* Node Version: CentOS = 8.9.3/whatever comes packaged with snapd for Ubuntu
* mongoDB Version: latest/whatever comes packaged with snapd

### Steps to Reproduce:

1. Login in as Admin
2. Go to "Custom Emoji"
3. Note that nowhere in the UI is there a button to add new emojis

### Expected behavior:

Fairly certain this was possible in the past...

### Actual behavior:
Only possible to search for custom emojis. Nowhere is there an option to add new ones.

### Relevant logs:
 N/A - logs remote nothing notable. No browser dev tool errors. It's as if the button that  should be there is out of frame.

Screen:
![screen](https://user-images.githubusercontent.com/29366220/40823278-ad98f050-6524-11e8-8260-3cbbdc6d52e3.png)

**Comment by ura14h** [comment]
I commented this issue at #10951

**Comment by TwizzyDizzy** [comment]
@rocket-cat label add "type: bug"

Can confirm this on 0.65.1 (tar deployment via ansible). Fun fact though. If you click an existing custom emoji to see details, the + in the sidebar appears.

At least in my case.

Cheers
Thomas

**Comment by rocketcrazy07** [comment]
Looking at the Dev log I noticed this error this happens when I try to manually add the "+" button as well.  Hope this might help somebody figure out what is wrong.
```
Exception in template helper: TypeError: Cannot read property 'rid' of undefined
    at https://our-server/111f6dfe3c5b59829f51d2620e189034953def3e.js?meteor_js_resource=true:431:69876
```

**Comment by vynmera** [comment]
This is fixed in #11049 @rocketcrazy07 :--)
